File: 1700. Number of Students Unable to Eat Lunch/solution.cpp
class Solution {
public:
    int countStudents(vector<int>& students, vector<int>& sandwiches) {
        int n=students.size();
        std::vector<int> arr = {0,0};
        for(int i=0;i<n;i++){
            arr[students[i]]+=1;
        }
        for(int j=0;j<n;j++){
            if(arr[sandwiches[j]]==0){
                break;
            }
            arr[sandwiches[j]]-=1;
        }
        return arr[0]+arr[1];
    }
};

File: 1700. Number of Students Unable to Eat Lunch/solution.py
class Solution:
    def countStudents(self, students: List[int], sandwiches: List[int]) -> int:
        count=0
        arr=[0,0]
        for i in students:
            arr[i]+=1
        for j in sandwiches:
            if arr[j]==0:
                break
            arr[j]-=1
        return sum(arr)
        